### PR TITLE
Fix grammatic error in site/en/tutorials/load_data/images.ipynb

### PR DESCRIPTION
**Pull request by erssebaggala**
Removes repeated words "to the"

### CODE REVIEW

**Review by MarkDaoust — DISMISSED** [review]
Thanks.

**Review by MarkDaoust — DISMISSED** [review]

File: site/en/tutorials/load_data/images.ipynb [hunk 1]
```diff
@@ -380,7 +380,8 @@
         "id": "2ZgIZeXaDUsF"
       },
       "source": [
-        "The `image_batch` is a tensor of the shape `(32, 180, 180, 3)`. This is a batch of 32 images of shape `180x180x3` (the last dimension referes to color channels RGB). The `label_batch` is a tensor of the shape `(32,)`, these are corresponding labels to the 32 images. \n"
+        "The `image_batch` is a tensor of the shape `(32, 180, 180, 3)`. This is a batch of 32 images of shape `180x180x3` (the last dimension referes to color channels RGB). The `label_batch` is a tensor of the shape `(32,)`, these are the corresponding labels to the
+        32 images. \n"
```
Comment (lamberta) [inline site/en/tutorials/load_data/images.ipynb:384]: I think this is breaking the JSON. Can you remove the extra newline? Thanks

Comment (erssebaggala) [reply site/en/tutorials/load_data/images.ipynb:384]: Let me fix it

**Review by lamberta — APPROVED** [review]
Thanks for the fix!

**Comment by 19-pragya** [comment]
can i work on this
??

**Pull request merged**